### DISCUSSION TITLE
Power on without cleaning cycle

### DISCUSSION THREAD

**Discussion by Sandst4** [TillFleisch/ESPHome-Philips-Smart-Coffee#52] [Q&A]
First of all, amazing what you have accomplished, thanks! I'm trying to connect a Philips 5400 as well, and I was wondering how you found out about the command to send to power on the machine without a cleaning cycle. As far as I'm aware the machines don't have any option like that on the display so I was wondering how you were able to find the command, thanks!

**Comment by TillFleisch** [comment]
I found out about the cleaning/non-cleaning power-on by logging the communication using the UART debug functionality.
The machine skips the cleaning cycle, if it has been pre-heated recently (the display unit issues the command, therefore it also decides when to perform cleaning).
I would recommend checking out the following resources regarding 5400:

#6

[Related home assistant community thread](https://community.home-assistant.io/t/esp-smart-offee-machine-philips-5400-series/568545)

**Comment by Sandst4** [comment]
Thanks! I'll have a look at it. I'll be waiting for some parts to be able to connect everything correctly.
